src/compute/tests/integration_tests.rs

```diff
@@ -173,10 +172,7 @@ async fn test_table_materialize() -> StreamResult<()> {
             Arc::new(StreamingMetrics::unused()),
             barrier_rx,
             system_params_manager.get_params(),
-            SourceCtrlOpts {
-                chunk_size: 1024,
-                rate_limit: None,
-            },
```

Comment:
I'd keep the struct because we may add other control params in the future.

Reply:
This PR does not remove this structure. In my mind this structure contains options that control a source reader, so I think it'll be better to construct this at the spots where readers are created.